src/scenes/home/termsOfService/termsOfService.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import Section from 'shared/components/section/section';
+
+class Terms extends Component {
+  render() {
+    return (
+      <div>
+        <Section title="Terms of Service" theme="white">
+          <p>
+            <i>Last updated: September 23, 2017.</i>
+          </p>
+          <p>
+            Please read these Terms of Service (&#34;Terms&#34;, &#34;Terms of Service&#34;) carefully before using the https:&#8725;&#8725;www.operationcode.org&#8725; website (the &#34;Service&#34;) operated by Operation Code (&#34;us&#34;, &#34;we&#34;, or &#34;our&#34;).
+          </p>
+          <p>
+            Your access to and use of the Service is conditioned upon your acceptance of and compliance with these Terms. These Terms apply to all visitors, users and others who wish to access or use the Service.
+          </p>
+          <p>
+          By accessing or using the Service you agree to be bound by these Terms. If you disagree with any part of the terms then you do not have permission to access the Service.
```

Comment:
This is nitpicky, but I will say it anyways. It appears that you are indenting some of your p tags, but not all. Please use proper indenting principles for your html 
```
<p>
  here is my content
</p>